github/libp2p.yml

```diff
@@ -469,6 +552,9 @@ repositories:
     default_branch: master
     description: "[DEPRECATED] Interfaces for securing libp2p connections; use
```

Comment:
@galargh Will adding the stale.yml files to deprecated repos cause the terraform apply to fail?

This also seems to contradict 
>adding stale.yml to all 226 non-archived repos in libp2p.

Reply:
Great catch! The script added the workflow to archived repos too because we're not tracking `archived` status in the config yet - https://github.com/libp2p/github-mgmt/blob/master/terraform/resources_override.tf#L15. I think we should, it'll also allow archiving repos through GitHub Management. Let me take care of this.

Reply:
All issues resolved! We started tracking `archived` property through GitHub Management and I rerun the fix workflow on this branch. We're now adding the workflow to 134 non-archived repos.